arch/common/timing_info_bench.c

```diff
@@ -43,13 +43,7 @@ u64_t arch_timing_value_swap_temp;
 #define TIMING_INFO_GET_TIMER_VALUE()  (TIMING_INFO_OS_GET_TIME())
 #define SUBTRACT_CLOCK_CYCLES(val)     (val)
 
-#elif defined(CONFIG_ARM64)
```

Comment:
is this covered in the #else now, @stephanosio ?

Reply:
@ioannisg Yes, `#elif defined(CONFIG_ARM64)` was a copy of the `#else` case, and the only reason we had that was to prevent ARM64 ending up in (what used to be) `#elif defined(CONFIG_ARM)` which relies on SysTick.